pkg/operator/operator.go

```diff
@@ -51,7 +54,30 @@ var (
 )
 
 func main() {
-	if err := config.Init(); err != nil {
+	config.Init()
+
+	config.AWS = aws.New(config.Cortex.Region, config.Cortex.Bucket)
```

Comment:
I think we could move lines 59 - 68 in `config.Init()`. `context.Init()` can still be here, but it would be nice for `config.Init()` to be self contained